Say Their Names/Source/Controller/Home/HomeController.swift

```diff
@@ -94,13 +98,16 @@ extension HomeController: UICollectionViewDataSource, UICollectionViewDelegateFl
     
     func collectionView(_ collectionView: UICollectionView, viewForSupplementaryElementOfKind kind: String, at indexPath: IndexPath) -> UICollectionReusableView {
         if collectionView.tag == 0 { return UICollectionReusableView() }
-        let headerView = peopleCollectionView.dequeueReusableSupplementaryView(ofKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: headerIdentifier, for: indexPath) as! PersonHeaderCell
+        let headerView = peopleCollectionView.dequeueReusableSupplementaryView(ofKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: headerIdentifier, for: indexPath) as! CarouselHeaderView
+        carouselDataResultsHandler = ResultsDataHandler(resultsData: self.carouselData)
+        headerView.resultsHandler = carouselDataResultsHandler
+        headerView.configure()
         return headerView
     }
     
     func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, referenceSizeForHeaderInSection section: Int) -> CGSize {
         let width = collectionView.frame.width - 32
-        return collectionView.tag == 0 ? .zero : .init(width: width, height: 170)
+        return collectionView.tag == 0 ? .zero : .init(width: width, height: 220)
```

Comment:
Double check to make sure the team hasn't already added a `Theme` or some other system that allows us to handle these magic numbers properly